ffi/tests/export_shared_fns.rs

```diff
@@ -2,9 +2,16 @@
 
 use std::{cmp::Ordering, mem::MaybeUninit};
 
-use iroha_ffi::{
-    def_ffi_fn, ffi_export, handles, FfiConvert, FfiOutPtrRead, FfiReturn, FfiType, Handle,
-};
+use iroha_ffi::{def_ffi_fns, ffi_export, FfiConvert, FfiOutPtrRead, FfiReturn, FfiType, Handle};
+
+iroha_ffi::handles! {FfiStruct1, FfiStruct2}
+
+def_ffi_fns! {""
```

Comment:
Maybe we can add something like `def_ffi_fns! { prefix: "...", ...}`? because at least for me it was not obvious what this do.

Reply:
done